themes/default/content/docs/pulumi-cloud/cloud-rest-api/_index.md

```diff
@@ -2735,15 +2735,16 @@ Search for resources belonging to the given organization.
 |------------|-------|---------------|----------|------------------------------------------------------------------------------------------------------------------------------------------|
-| org        | path  | string        | true     | Name of the organization to search.                                                                                                      |
-| query      | query | string        | false    | The search query to execute. If omitted all resources are returned (subject to any pagination limits).                                   |
-| sort       | query | array[string] | false    | Results are returned sorted by this field value.                                                                                         |
-| asc        | query | boolean       | false    | Whether to return results in ascending or descending sort order.                                                                         |
-| size       | query | integer       | false    | How many results to return at a time.                                                                                                    |
-| page       | query | number        | false    | The page of results to return.                                                                                                           |
-| continue   | query | string        | false    | A continuation token for pagination.                                                                                                     |
-| facet      | query | array[string] | false    | If provided, an aggregation will be returned with the top-5 values for the given facet, along with how many resources have those values. |
-| properties | query | boolean       | false    | Whether to include resource properties in results. Not supported for all subscriptions.                                                  |
+| org        | path  | string           | true     | Name of the organization to search.                                                                                                      |
+| query      | query | string           | false    | The search query to execute. If omitted all resources are returned (subject to any pagination limits).                                   |
+| sort       | query | array[string]    | false    | Results are returned sorted by this field value.                                                                                         |
+| asc        | query | boolean          | false    | Whether to return results in ascending or descending sort order.                                                                         |
+| size       | query | integer          | false    | How many results to return at a time.                                                                                                    |
+| paginate   | query | "cursor", "page" | false    | If requesting multiple pages of results, the pagination method to use.
```

Comment:
We should indicate that the default value is `page`

Reply:
We should probably make this `string` and indicate that valid values are `cursor` and `page`.

Reply:
[152f60b](https://github.com/pulumi/pulumi-hugo/pull/3105/commits/152f60b25d5c2cfd937b87f3a16207f9444ae711)